### PR TITLE
Reconcile templated config with the configuration from the web UI

### PR DESCRIPTION
**Pull request by nuclearsandwich**
This updates the configuration templates with what is currently in use after the pluigin upgrades last night.

There are some lingering whitespace artifacts that still appear during  a dry run but running it once should clear those up.

<details><code><pre>
Connecting to Jenkins 'http://ci.ros2.org'
Connected to Jenkins version '2.46.1'
Updating job 'ci_linux' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -168 +168 @@
    -        </script>
    +          </script>
    >>>
Updating job 'packaging_linux' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -127 +127 @@
    -        </script>
    +          </script>
    @@ -155 +155 @@
    -  case $CI_AMENT_TEST_ARGS in 
    +  case $CI_AMENT_TEST_ARGS in
    >>>
Updating job 'nightly_linux_debug' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_linux_coverage' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_linux_release' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_linux_repeated' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'ci_linux-aarch64' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -168 +168 @@
    -        </script>
    +          </script>
    >>>
Updating job 'packaging_linux-aarch64' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -127 +127 @@
    -        </script>
    +          </script>
    @@ -155 +155 @@
    -  case $CI_AMENT_TEST_ARGS in 
    +  case $CI_AMENT_TEST_ARGS in
    >>>
Updating job 'nightly_linux-aarch64_debug' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_linux-aarch64_release' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_linux-aarch64_repeated' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'ci_osx' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -168 +168 @@
    -        </script>
    +          </script>
    >>>
Updating job 'packaging_osx' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -127 +127 @@
    -        </script>
    +          </script>
    @@ -155 +155 @@
    -  case $CI_AMENT_TEST_ARGS in 
    +  case $CI_AMENT_TEST_ARGS in
    >>>
Updating job 'nightly_osx_debug' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_osx_release' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_osx_repeated' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -172 +172 @@
    -        </script>
    +          </script>
    >>>
Updating job 'ci_windows' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -160 +160 @@
    -        </script>
    +          </script>
    >>>
Updating job 'packaging_windows' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -119 +119 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_win_deb' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -164 +164 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_win_rel' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -164 +164 @@
    -        </script>
    +          </script>
    >>>
Updating job 'nightly_win_rep' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -164 +164 @@
    -        </script>
    +          </script>
    >>>
Updating job 'ci_launcher' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -122,0 +123 @@
    +
    @@ -143,0 +145 @@
    +
    @@ -156,0 +159 @@
    +
    @@ -169,0 +173 @@
    +
    >>>
Updating job 'ci_turtlebot-demo' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -168 +168 @@
    -        </script>
    +          </script>
    >>>

</pre></code></details>

cc @tfoote in case this helps with the ROS buildfarm updates.

### CODE REVIEW

**Comment by dirk-thomas** [comment]
There should be no diff at all. Otherwise the configuration will always flip-flop when being configure using the web ui vs. the remote API.

**Comment by nuclearsandwich** [comment]
> There should be no diff at all. Otherwise the configuration will always flip-flop when being configure using the web ui vs. the remote API.

Okay, the only diff now is a whitespace fixup analogous to one I performed when updating the ci_job template for #32.

<details><code><pre>
(.virtualenv) steven@steven-desktop:~/osrf/ros2ci$ ./create_jenkins_job.py 
Connecting to Jenkins 'http://ci.ros2.org'
Connected to Jenkins version '2.46.1'
Skipped 'ci_linux' because the config is the same
Updating job 'packaging_linux' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -155 +155 @@
    -  case $CI_AMENT_TEST_ARGS in 
    +  case $CI_AMENT_TEST_ARGS in
    >>>
Skipped 'nightly_linux_debug' because the config is the same
Skipped 'nightly_linux_coverage' because the config is the same
Skipped 'nightly_linux_release' because the config is the same
Skipped 'nightly_linux_repeated' because the config is the same
Skipped 'ci_linux-aarch64' because the config is the same
Updating job 'packaging_linux-aarch64' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -155 +155 @@
    -  case $CI_AMENT_TEST_ARGS in 
    +  case $CI_AMENT_TEST_ARGS in
    >>>
Skipped 'nightly_linux-aarch64_debug' because the config is the same
Skipped 'nightly_linux-aarch64_release' because the config is the same
Skipped 'nightly_linux-aarch64_repeated' because the config is the same
Skipped 'ci_osx' because the config is the same
Updating job 'packaging_osx' (dry run)
    <<<
    --- remote config
    +++ new config
    @@ -155 +155 @@
    -  case $CI_AMENT_TEST_ARGS in 
    +  case $CI_AMENT_TEST_ARGS in
    >>>
Skipped 'nightly_osx_debug' because the config is the same
Skipped 'nightly_osx_release' because the config is the same
Skipped 'nightly_osx_repeated' because the config is the same
Skipped 'ci_windows' because the config is the same
Skipped 'packaging_windows' because the config is the same
Skipped 'nightly_win_deb' because the config is the same
Skipped 'nightly_win_rel' because the config is the same
Skipped 'nightly_win_rep' because the config is the same
Skipped 'ci_launcher' because the config is the same
Skipped 'ci_turtlebot-demo' because the config is the same
</pre></code></details>

File: job_templates/ci_launcher_job.xml.em [hunk 1]
```diff
@@ -54,11 +54,12 @@
                 </hudson.plugins.parameterizedtrigger.BooleanParameterConfig>
               </configs>
             </hudson.plugins.parameterizedtrigger.BooleanParameters>
-@[end if]
+@[end if]@
```
Comment (dirk-thomas) [inline job_templates/ci_launcher_job.xml.em]: Looks like this was introduced in #47.

**Comment by nuclearsandwich** [comment]
Committed the whitespace fixup to CI. Should be no spurious changes the next time anyone updates.

**Pull request merged**